qmi/instruments/newport/tlb670x.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, context: QMI_Context, name: str, serial_number: Union[int, st
 
         # Prepare handle to DLL and device; set in open().
         self._handle: Any = ctypes.WinDLL(self.DLL_NAME)
-        self._device_id: int = 0
+        self._device_id: int = -1  # Initialize with non-existing device number. The real values are in range [0-31].
```

Comment:
why this change?

Reply:
For this: https://github.com/QuTech-Delft/QMI/pull/40/files#diff-2223e03ca4c943a12b3649a6bc0f13631a91ed2278392d664c9544e387d07469R209